ClientSide(FrontEnd)/clay_/src/components/signup.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
use one import statement for importing react and its hooks

Reply:
i removed extra hooks from that signup page